extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,631 @@
+
+State information for the tileset is stored in the following bitstreams:
+
+
+| Bitstream | Description       | Size (bits) |
+|------|-------------------|---|
+|  subdivision  | Encodes subdivision of each tile | 2 |
+|  content  | Encodes availability of content for tile | 1 |
+|  metadata  | Encodes availability of metadata for tile | 1 |
+
+#### Subdivision
+
+The subdivision state for each tile determines if and how it subdivides into children tiles, as per the `tilingScheme`. The subdivision state is encoded in 2 bits, and padded with 0s at the end to meet byte boundaries. The jump buffer generation will ignore the padding. Subdivision in a tile can have one of the following states:
+
+| Code | Description                                            |
+|------|--------------------------------------------------------|
+| `00` | Does not subdivide.                                    |
+| `01` | Subdivides into external tileset at implicit location. | 
+| `10` | Subdivides into external tileset at explicit location. |
+| `11` | Subdivides internally.                                 |
+
+When a tile divides externally, the content and metadata for the root tile are obtained from the external tileset.
+##### External Tileset at Implicit Location
+
+An external tileset may exist within the file structure of its parent tileset, with the root of the external tileset being present at the implicit location of the tile with subdivision state `01`.
+
+##### External Tileset at Implicit Location
+
+An external tileset may exist outside the file structure of its parent tileset, with the root of the external tileset being present in the `tile.json` at the implicit location of the tile with subdivision state `10`.
+
+##### Complete Levels
+
+For tilesets that have uniform subdivision for each tile up to a certain level, it is redundant to store the tile subdivision state information. The `completeLevels` property is used to indicate how many levels within the tileset are complete. Complete levels are levels in which all tiles subdivide implicitly and internally (using the bitcode `11`).
+
+#### Content
+
+This is a one bit representation of whether or not a tile has content associated with it.
+
+| Code | Description       |
+|------|-------------------|
+| `0`  | No content        |
+| `1`  | Has content       |
+
+When the tile has content, the content payload can be found implicitly by combining the tile location with the tile extension.
+
+#### Metadata
+
+This is a one bit representation of whether or not a tile has content associated with it.
+
+| Code | Description       |
+|------|-------------------|
+| `0`  | No metadata       |
+| `1`  | Has metadata      |
+
+
+When the tile has content, the index of the tile in the subdivision buffer will be the index of corresponding metadata property in the associated buffer in the `3DTILES_tile_metadata` extension.
```

Comment:
This doesn't leave room for empty tiles having metadata, which is something worth supporting. In that case the index would be the number of tiles with metadata before it.

Reply:
This was the result of a typo. Content and metadata are independent concepts.